chassis/util/src/main/java/com/griddynamics/jagger/util/StandardMetricsNamesUtil.java

```diff
@@ -71,6 +71,49 @@
     public static final String DURATION_OLD_ID = "duration-old";
     public static final String ITERATION_SAMPLES_OLD_ID = "samples-old";
 
+
+
+    // standard monitoring metric names
+    public static final String MON_CPULA_1 = "mon_cpula_1";
```

Comment:
they are defined in com.griddynamics.jagger.dbapi.parameter.DefaultMonitoringParameters